Document/0x08-Testing-Tools.md

```diff
@@ -66,6 +70,20 @@ To perform security testing different tools are available in order to be able to
 - [Cydia Substrate Module: Android SSL Trust Killer](https://github.com/iSECPartners/Android-SSL-TrustKiller) - Blackbox tool to bypass SSL certificate pinning for most applications running on a device.
 - [Cydia Substrate Module: RootCoak Plus](https://github.com/devadvance/rootcloakplus) - Patch root checking for commonly known indications of root.
 - [Android-ssl-bypass](https://github.com/iSECPartners/android-ssl-bypass) - an Android debugging tool that can be used for bypassing SSL, even when certificate pinning is implemented, as well as other debugging tasks. The tool runs as an interactive console.
+- [Frida CodeShare](https://codeshare.frida.re/) - The Frida CodeShare project is comprised of developers from around the world working together with one goal - push Frida to its limits in new and innovative ways.
+
+#### Security Libraries
+
+- [PublicKey Pinning](https://www.owasp.org/index.php/Certificate_and_Public_Key_Pinning#Android) - Pinning in Android can be accomplished through a custom X509TrustManager. X509TrustManager should perform the customary X509 checks in addition to performing the pinning configuration.
+- [Android Pinning](https://github.com/moxie0/AndroidPinning) - A standalone library project for certificate pinning on Android.

```

Comment:
let's remove this as it is outdated. the way to do this is based on okhttp nowadays...